build-binutils.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019 The ClangBuiltLinux Authors
+# Description: Builds a standalone copy of binutils
+
+import argparse
+import multiprocessing
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import utils
+
+
+# Convert the host architecture to the first part of a target triple
```

Comment:
I think in Python it's preferred to use `""" Comment """` so that you can run `help(fn)` in the repl?

Reply:
Harsh did this for all the scripts.